pkgs/top-level/python-packages.nix

```diff
@@ -20338,6 +20357,25 @@ in {
     };
   };
 
+  pyGithub = buildPythonPackage rec {
+    name = "pyGithub-${version}";
+    version = "1.32";
+    src = pkgs.fetchurl {
+      url = "https://github.com/PyGithub/PyGithub/archive/v${version}.tar.gz";
+      sha256 = "0gip9ksm0m78wd2rhv4ahdaclw5fqz1f7bpdv8y3zpq7i9crf443";
+    };
+    propagatedBuildInputs = with self; [ python-jose ];
+    buildInput = with pkgs; [ openssl pkgconfig ];
```

Comment:
You're right. Anyway, both openssl and pkgconfig are found to be unnecessary so I removed the line. I do not remember why I put them in.